aws/v12.1.1/README.md

```diff
@@ -0,0 +1,19 @@
+# :zap: Giant Swarm Release v12.1.1 for AWS :zap:
+
+<< Add description here >>
+
+## Change details
+
+
+### cert-manager [2.0.2](https://github.com/giantswarm/cert-manager-app/releases/tag/v2.0.2)
+
+#### Changed
+- Install CRDs during chart installation only. ([#46](https://github.com/giantswarm/cert-manager-app/pull/46))
+
+
+
+### kiam [1.3.2](https://github.com/giantswarm/kiam-app/releases/tag/v1.3.2)
+
+Not found
```

Comment:
Needs updating to 1.4.0 and changelog entry

Reply:
already done ;)